english/breakfast/breakfast.md

```diff
@@ -0,0 +1,4 @@
+You see a pot of coffee near you and you realize that you are hungry. Well, you poured coffee in a mug and picked some 
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.